tests/testthat/_snaps/bayes.md

```diff
@@ -57,38 +61,41 @@
 
     Code
       tune:::check_bayes_initial_size(5, 3, FALSE)
-    Message <simpleMessage>
+    Message
       ! There are 5 tuning parameters and 3 grid points were requested. This is likely to cause numerical issues in the
         first few search iterations.
 
 ---
 
     Code
       tune:::check_bayes_initial_size(5, 3, TRUE)
-    Message <simpleMessage>
+    Message
       ! There are 5 tuning parameters and 3 grid points were requested. This is likely to cause numerical issues in the
         first few search iterations. With racing, only completely resampled parameters are used.
 
 ---
 
     Code
       tune:::check_bayes_initial_size(5, 1, FALSE)
-    Error <rlang_error>
-      ! There are 5 tuning parameters and a single grid point was requested. The GP model requires 2+ initial points but there should be more initial points than there are tuning paramters. 
+    Condition
+      Error in `tune:::check_bayes_initial_size()`:
+      ! ! There are 5 tuning parameters and a single grid point was requested. The GP model requires 2+ initial points but there should be more initial points than there are tuning paramters. 
```

Comment:
Just want to highlight how the errors look now @topepo; we probably want to address this.

Reply:
Do you mean the function called back? I'd leave as-is for this PR and we can add an issue to improve it overall.